### PR TITLE
Added onUnitFocus/Blur and tab functionality.

### PR DESCRIPTION
**Pull request by yidah**
Hi Nik,

I noticed that running **npm update** added some changes to the **package-lock.json** file although I did not add any dependency.

Also I left the input we added as a way to test the tab functionality but I am happy to remove it.

The **onUnitFocus** and **onUnitBlur** function arguments in the DurationControl.**stories**.tsx write on the console, in order to test these events in the DurationControl. I was not sure how to make them dynamic like when you enter text or boolean values.

### CODE REVIEW

File: stories/DurationControl.stories.tsx [hunk 1]
```diff
@@ -1,45 +1,65 @@
...
-  disabled: false,
-  hideSpinner: false,
-  label: ""
-};
\ No newline at end of file
+	pattern: "Days {dd} Hours {hh} Minutes {mm} Seconds {ss} Millis {fff}",
+	disabled: false,
+	hideSpinner: false,
+	label: "",
+	onUnitFocus: (unit) => console.log("focus", unit),
```
Comment (nikkorn) [inline stories/DurationControl.stories.tsx:63]: Remove onUnitFocus and onUnitBlur from here

File: stories/DurationControl.stories.tsx [hunk 2]
```diff
@@ -1,45 +1,65 @@
...
-            size={60}
-            type="number"
-            value={millis}
-            onChange={(event) => setMillis(parseInt(event.target.value))}
-        ></input>
-    </div>
-  );
-}
+	return (
+		<div>
```
Comment (nikkorn) [inline stories/DurationControl.stories.tsx:17]: This return value shouldn't have changed (apart from you adding the onUnitFocus and onUnitBlur callbacks which should call console.log)

**Review by nikkorn — COMMENTED** [review]
Looks really good, just a couple of comments

File: stories/DurationControl.stories.tsx (hunk 2 above)
Comment (yidah) [reply stories/DurationControl.stories.tsx:10]: Back as it was before 🙂.

**Review by nikkorn — APPROVED** [review]
Fantastic stuff

**Pull request merged**